handlebars-cmd/plan.sh

```diff
@@ -0,0 +1,62 @@
+pkg_name=handlebars-cmd
+pkg_origin=core
+pkg_version="0.1.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source=nosuchfile.tar.gz
+pkg_deps=(
+  core/coreutils
+  core/node
+)
+pkg_build_deps=(
```

Comment:
If you already have something in `pkg_deps` you don't need it in `pkg_build_deps`.